Source/VSIX/AllRelativityTemplates/source.extension.vsixmanifest

```diff
@@ -119,6 +119,7 @@
         <Asset Type="File" d:Source="File" Path="packages\Relativity.Other.10.3.191.8.nupkg" />
         <Asset Type="jQuery.Validation.1.19.3.nupkg" d:Source="File" Path="packages\jQuery.Validation.1.19.3.nupkg" d:VsixSubPath="packages" />
         <Asset Type="jQuery.3.6.0.nupkg" d:Source="File" Path="packages\jQuery.3.6.0.nupkg" d:VsixSubPath="Packages" />
+        <Asset d:Source="File" Path="packages\Newtonsoft.Json.13.0.2.nupkg" Type="File" />
```

Comment:
should we remove 12.x and 8.0.3 from this file?

Reply:
Yeah I'll try to clean it up, will also make the file size a bit smaller too

Reply:
Done and tested